packages/dcos-integration-test/extra/test_checks.py

```diff
@@ -59,7 +59,14 @@ def test_checks_cli(dcos_api_session):
 
 
 def test_checks_api(dcos_api_session):
-    """Test the checks API at /system/checks/"""
+    """
+    Test the checks API at /system/checks/
+    This will only test the checks API can be successfully reached on a real
+    cluster and that the checks it returns when running GET will match those it
+    reports when running the checks with POST. It does not check the results
+    status as `dcos_api_session` will wait while being built for the cluster to
+    be running so we won't expect any checks here to fail anyway.
```

Comment:
This docstring is out of date after your latest update, which restores the status check.